app/setup_handlers.go

```diff
@@ -32,6 +33,12 @@ func GetDefaultUpgradeHandlerVersion() string {
 func SetupHandlers(app *App) {
 	allUpgrades := upgradeTracker{
 		upgrades: []upgradeTrackerItem{
+			{
+				index: 1752528615,
```

Comment:
Why this number?

Reply:
we need timestamp i think for ordering tracker items, not sure which one so i took the current one, will double check